requirements.txt

```diff
@@ -22,6 +22,6 @@ six==1.11.0
 SQLAlchemy==1.2.10
 thespian==3.9.2
 Werkzeug==0.14.1
-PyYAML==4.2b1
+PyYAML
```

Comment:
If I am not wrong this just installs the latest version. Are we supposed to do that? We can have it set to the latest version of the package on which our repo works. i.e. v5.1

Reply:
this is fine as we automatically get all the security updates